jupyter_server/pytest_plugin.py

```diff
@@ -62,6 +63,11 @@ def mkdir(tmp_path, *parts):
 env_config_path = pytest.fixture(
     lambda tmp_path: mkdir(tmp_path, "env", "etc", "jupyter")
 )
+some_resource = u"The very model of a modern major general"
```

Comment:
Nice Hamilton quote. 😉

Reply:
Not sure here... I have merge master and they are still there... I can only say it is not me as I do not know Hamilton...